### ISSUE TITLE
[IC2/TiCo] Compatibility with 1.7.10?

### ISSUE THREAD

**Issue by Revemohl** [CyanideX/Unity#74]
I've noticed that Unity's recent updates for these two mods are completely incompatible with MC 1.7.10. I thought fixing them all would be a simply matter of just renaming everything, but quickly found a few textures for IC2 that are stitched together on 1.7.10 but separated on 1.10, like every generator or machine for example. 

So, would it be possible to release an 1.7.10-compatible updated version of Unity, for these two mods at least, possibly when the next full version is complete? Yeah 1.7.10 is old and all but it still has plenty of life left, and as far as I know there's a decent amount of people still playing on it.

**Comment by Katalliaan** [comment]
Probably the best/easiest solution would be to create a 1.7.10 branch. I do have a version of the 1.7.10 creative inventory GUI that I made based on CyanideX's modifications to the 1.9 version (which moved the armor slots around) that I could contribute.

**Comment by Yulife** [comment]
For me 1.7.10 is dead development wise. Dunno about @CyanideX though, there are more sensible things to spend times on

**Comment by BBoldt** [comment]
We should have had separate branches in the first place. Separating them now might be a bit complicated,  but I'll see what I can do.

**Comment by Yulife** [comment]
Well, whats the next version on the horizont? :stuck_out_tongue:

**Comment by CyanideX** [comment]
There are already two branches and I will be pushing the 1.7.10 version of IC2 to that separate branch. After that we will no longer be supporting older versions of MC/mods.

**Comment by CyanideX** [comment]
Unfortunately, Tinker's will not be for 1.7.10.

**Comment by Revemohl** [comment]
I took a look at TCon's textures and these ones do in fact seem to be a matter of just needing to be renamed, but at the same time it looks like there are plenty more of them for 1.7.10 than there are for 1.10 (even taking into account how the Unity TCon textures are WIP), and tools seem to follow a completely different texture structure as well so I don't mind that.

**Comment by CyanideX** [comment]
[Yeah, they went from over 3000 textures in 1.7.10 to just over 500.](https://twitter.com/bonusboni/status/768034840214200325)

**Comment by CyanideX** [comment]
I've decided we're only going to support 1.10.2 from her on out.

I will release my 1.7.10 IC2 retexture as a separate download on CurseForge but it's too much trouble to maintain two branches.

**Comment by Yulife** [comment]
Aye. If 1.11 ever comes up, we will make sure to make it a seperate branch

**Comment by Revemohl** [comment]
Here, I finally decided to start a map with IC2 and felt like porting the textures by myself instead of waiting for @CyanideX.
https://www.dropbox.com/s/9977c36k2is36ny/Unity%20IC2%201.7.10%20backport.zip?dl=0
I had a few issues, though:
- Couldn't find any of these files among Unity's textures: itemLathingTool.png, itemTurningBlanksIRON.png, itemTurningBlanksWOOD.png, reactorVentCore.png, turnables/IRON, turnables/WOOD
- Couldn't find matching IC2 files for these Unity textures: iridium_reflector.png, quantum armor overlays obscurator_mask.png, rci_lzh/rsh machines, fluid tank
- Reactor components may be wrong
- Couldn't find armor models, so they're ugly for now
- Terraformer isn't animated because that would heavily inflate the texture size
- blockPersonalChest.png and Unity's personal chest are different, so I didn't touch them
- Lathe and Fermenter machines are missing from the Unity side

I'd be thankful if someone reviewed at least these points, since old IC2's naming scheme was a catastrophe so I had to go with pure guesswork on the reactor parts. I also have absolutely no idea how to submit a pull request, so I'll leave that for someone more capable if they feel like it.
I also wonder if it wouldn't just be better to directly put this on Curseforge after fixing whatever needs to be fixed. Everything else seems to be doing fine enough according to the quick check I did.
